[SC-167] Wyświetlenie informacji o dodatkowych punktach u studenta

dmncp opened the pull request:
Uznałem, że dodam tabsy w widoku punktów, bo jak dałem tabelę pod tabelą to pojawiają się bugi z sidebarem - np sidebar mimo że jest fixed to jeździ jak mu się podoba (Tych bugów uwierz, że nie chcemy teraz rozwiązywać. Lepiej po prostu "ukryć" te problemy takimi tabsami). Co do samej tabeli to dałem tam takie info jakie uznałem za najbardziej przydatne, ale jak coś to można dodać inne kolumny ;)

dmncp committed:
[SC-165] generalization of json editing component and using it to edit activity

dmncp committed:
[SC-166] delete activity modal

dmncp committed:
[SC-152] modal for adding free points

dmncp committed:
[SC-167] gratis points student view - table inside tabs

kwdrt commented:
Odnośnie tabsów - imo fajnie zobaczyć co dostało się dodatkowo oddzielnie nawet lepiej + unikamy baaardzo długiego scrolla gdyby gra miała dziesięć tysięcy aktywności, myślę że ta zmiana to po prostu dobry UX

kwdrt commented on frontend/src/components/student/PointsPage/Points.js:
```diff
@@ -28,8 +30,19 @@ export default function Points() {
+            <LastPointsTable />
+          </Tab>
+          <Tab eventKey={'gratis-points'} title={'Punkty dodatkowe'}>
+            <GratisPointsTable />
```
Trochę średnio-angielsko jest 'Gratis' (chociaż swoją drogą chyba z łaciny, więc też popularny język swego czasu w Europie XD) - myślę że lepszym odpowiednikiem jest np. 'Bonus'

kwdrt reviewed (approved):
Jeden non-block dotyczący nazewnictwa, ale uważam że wygląda super tak jak jest, a że chowamy przy tym coś związanego z RWD to nawet bonus 🎁

kwdrt commented:
Mam drobną prośbę - jeśli robisz rzeczy które nie wymagają czegoś co jest częścią innego PR (np. chyba tutaj) to staraj się odbijać od develop - inaczej jest z 29 plików do przejrzenia i trochę ciężej się sprawdza

Wiem że trochę moja wina że nie zdążyłem przerobić poprzednich, ale na przyszłość tylko zostawiam

dmncp committed:
[SC-167] Merge remote-tracking branch 'origin/develop' into SC-167-wyswietlenie-informacji-o-dodatkowych-punktach-u-studenta

# Conflicts:
#	frontend/src/components/professor/ChapterDetails/ChapterDetails.js
#	frontend/src/components/professor/ChapterDetails/ChapterDetailsStyles.js
#	frontend/src/components/professor/GroupsPage/Table/GroupsTable.js

dmncp committed:
[SC-167] bonus points functions rename
